Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -122,19 +121,19 @@ import PayPalCheckout
                         case .vault:
                             action.set(billingAgreementToken: order.orderID)
                         @unknown default:
+                            self?.apiClient.sendAnalyticsEvent(BTPayPalNativeCheckoutAnalytics.tokenizeFailed)
```

Comment:
```suggestion
```
🧶 : This seems like a merchant failure and not necessarily a failure with the Gateway's tokenization service. Plus it wasn't there before I'd say it can be removed.

Reply:
We had a discussion that `tokenize:started event` needs to conclude with an end event` succeeded/failed/canceled`.  We are viewing the whole process as tokenize and completion to the merchant app needs to return one of the end events.

Reply:
Oh interesting yeah. It looks like this has more to do with a merchant error than a gateway error. I wanted to call it out. Take or leave!

Reply:
Yeh -  we should consider sending additional error details as metadata on the event in follow-ups.

The thought was to start with streamlined/simplified analytic events as much as possible, that tell a good conversion story, and then iterate from there.